Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -642,14 +640,24 @@ static wxColour blend50(const wxColour& c1, const wxColour& c2)
   return a << 24 | b << 16 | g << 8 | r;
 }
 
-void CGameListCtrl::SetBackgroundColor()
+static wxColour ContrastText(const wxColour& bgc)
+{
+  int lum = (((0.299 * bgc.Red()) + ((0.587 * bgc.Green()) + (0.114 * bgc.Blue()))));
+  return (lum > LUM_THRESHOLD) ? *wxBLACK : *wxWHITE;
+}
+
+void CGameListCtrl::SetColors()
 {
   for (long i = 0; i < GetItemCount(); i++)
   {
     wxColour color = (i & 1) ? blend50(wxSystemSettings::GetColour(wxSYS_COLOUR_3DLIGHT),
                                        wxSystemSettings::GetColour(wxSYS_COLOUR_WINDOW)) :
                                wxSystemSettings::GetColour(wxSYS_COLOUR_WINDOW);
     CGameListCtrl::SetItemBackgroundColour(i, color);
+    if (SConfig::GetInstance().m_ColorCompressed && m_ISOFiles[i]->IsCompressed())
+      SetItemTextColour(i, wxBLUE);
```

Comment:
I'd fixed the *wxBLUE, I don't know how that got lost.  Anyway, hopefully it's all good now...